Content.Server/Disease/Components/DiseaseCarrierComponent.cs

```diff
@@ -0,0 +1,35 @@
+using System.Linq;
+using Content.Shared.Disease;
+
+namespace Content.Server.Disease.Components
+{
+    [RegisterComponent]
+    /// <summary>
+    /// Allows the enity to be infected with diseases.
+    /// Please use only on mobs.
+    /// </summary>
```

Comment:
attributes should go under xmldocs. same goes for a couple other places in this file

Reply:
as well as like 500 other places, just fix them all im not gonna put reviews for the rest